spec/features/cannot_peep_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+feature 'cant peep if not signed in' do
+
+  before(:each) do
+    user = User.create(name: 'name',
+    username: 'user',
+    email: 'test@test.com',
+    password: 'password',
+    password_conf: 'password')
+
+    user.peeps.create(message: 'Hello',
+    name: 'name',
+    username: 'user')
+
+    sign_in('test@test.com', 'password')
+  end
+
+  scenario 'cant view page' do
+    visit ('/peeps')
+    click_button 'Sign Out'
+    visit ('/peeps')
+    expect(page).to have_current_path('/')
+  end
+  scenario 'error message' do
```

Comment:
I'd struggle to get a clear overview of the system if I ran these specs (that's how I use specs: like documentation). Is there some way you could rewrite the scenarios to be more descriptive when they document out to the command-line?